core/templates/base-components/warnings-and-alerts.component.html

```diff
@@ -21,6 +21,7 @@
   .oppia-toast-warning {
     background-color: #f89406;
     background-image: none;
+    font-weight: bold;
```

Comment:
Can we discuss / do this as part of a separate PR? We really want minimal changes that fix the actual bug so it can be cherrypicked into the release branch.